libs/ledger-live-common/src/families/bitcoin/specs.ts

```diff
@@ -325,10 +325,10 @@ const bitcoin: AppSpec<Transaction> = {
 const bitcoinTestnet: AppSpec<Transaction> = {
   name: "Bitcoin Testnet",
   currency: getCryptoCurrencyById("bitcoin_testnet"),
-  dependency: "Bitcoin",
   appQuery: {
     model: DeviceModelId.nanoS,
     appName: "Bitcoin Test",
+    appVersion: "app_2.0.99-prerelease",
```

Comment:
Have you verified that the bot runs actually use these custom versions?
I think by default the bot ignores all coin apps with a custom version, I think forcing it like like this works, but better confirm by checking the bot report.